.github/workflows/api-gateway-prod-worker.yml

```
@@ -9,12 +9,15 @@ on:
         required: true
       JFP_AWS_SECRET_ACCESS_KEY:
         required: true
+permissions:
+  contents: read
 
 jobs:
   build-and-deploy:
     environment: Production
     env:
       IMAGE_TAG: ${{ github.sha }}
+      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
```

actionlint (1.7.7)

property "nx_cloud_access_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; aws_default_region: string; github_token: string; jfp_aws_access_key_id: string; jfp_aws_secret_access_key: string}

(expression)